BinaryEncoding.md

```diff
@@ -3,120 +3,143 @@
 This document describes the [portable](Portability.md) binary encoding of the
 [Abstract Syntax Tree](AstSemantics.md) nodes.
 
-The binary encoding is designed to allow fast startup, which includes reducing
-download size and allow for quick decoding. For more information, see the 
-[rationale document](Rationale.md#why-a-binary-encoding)
-
-Reducing download size, is achieved through three layers:
-
- * The **raw** binary encoding itself, natively decoded by the browser, and to
-   be standardized in the [MVP](MVP.md).
- * **Specific** compression to the binary encoding, that is unreasonable to
-   expect a generic compression algorithm like gzip to achieve.
-   * This is not meant to be standardized, at least not initially, as it can be
```

Comment:
We need to spec layer 1 in the MVP or it will not be V, Viable. We can defer layer 2 as much as we want because it's much more reasonable to do it in user-space. (Technically apache/nginx content-encoding gzip would count as an implementation of layer 2.)